route/build/src/read_control.f90

```diff
@@ -585,6 +586,38 @@ SUBROUTINE read_control(ctl_fname, err, message)
    end select
  end do
 
+ ! Control routing method dependent variable name - routedRunoff, volume, elevation etc.
+ if ((nRoutes<=2 .and. (routeMethods(1)==0 .or. routeMethods(2)==0)) .and. trim(lower(outputNameOption))=='generic') then
```

Comment:
`(routeMethods(1)==0 .or. routeMethods(2)==0))` should be `any(routeMethods==0)`

Reply:
this should be like this
```
 isGeneric = trim(lower(outputNameOption))=='generic'
 onlyOneRouting = ((nRoutes==2 .and. any(routeMethods==accumRunoff) .or. nRoutes==1))
```